### DISCUSSION TITLE
[TPA] Dudas entrega 3

### DISCUSSION THREAD

**Discussion by BrianLuna** [dds-jv/foro#146] [TPA - 2020] [answered]
Buenas noches, luego de leer el enunciado de las entrega N°3 nos surgen las siguientes dudas:

Respecto del reporte mensual de egresos: Se espera que implementemos ese reporte?

Respecto de las etiquetas de los egresos: Se espera que los egresos puedan tener más de una etiqueta? En ese caso, cómo se mostrarían en el reporte? Por ejemplo, si la entidad compra una mesa y le ponen la etiqueta de Amoblamiento (Tipo de producto) y Pepito (proveedor), ese monto aparecería dos veces en el reporte ¿?

Respecto del comportamiento de las categorías de entidades: Uno de los puntos dice: 
`No permitir que una entidad base pueda ser parte de alguna entidad jurídica.`
Eso se refiere a que una entidad base no pueda ser parte de una entidad jurídica o de ninguna de ellas?

Muchas gracias

**Comment by gastonprieto** [comment, marked as answer]
Buenas!

> Respecto del reporte mensual de egresos: Se espera que implementemos ese reporte?

Si, pero la idea es que tengas los objetos y un mensaje de dominio como para generar esa información. 

> espera que los egresos puedan tener más de una etiqueta? 

Podrían, pero si lo hacen con una etiqueta también esta bien.

> En ese caso, cómo se mostrarían en el reporte

Estarían los gastos repetidos no habría problema

> Eso se refiere a que una entidad base no pueda ser parte de una entidad jurídica o de ninguna de ellas?

no entendi. La idea de esto es evitar que cuando tenga configurado ese comportamiento, no se pueda tener a los objetos configurados de esta forma.

**Comment by BrianLuna** [comment]
Muchas gracias @gastonprieto
